testing/ios/IosUnitTests/Tests/FlutterEngineConfig.xcconfig

```diff
@@ -1,2 +1,3 @@
 FLUTTER_ENGINE[arch=x86_64]=ios_debug_sim_unopt
 FLUTTER_ENGINE[arch=arm64]=ios_debug_sim_unopt_arm64
+FLUTTER_ENGINE=ios_debug_sim_unopt
```

Comment:
@jmagman Not sure if you want to keep this or fix it in a separate PR?

Reply:
Bleh may as well keep it, my attempt to do the right thing based on architecture was totally thwarted by Xcode passing in `ARCH=unknown` at certain build steps.